namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-management-preprod/resources/elasticache.tf

```diff
@@ -0,0 +1,28 @@
+module "ec-cluster-offender-management-allocation-manager" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=3.1"
+  cluster_name           = "${var.cluster_name}"
+  cluster_state_bucket   = "${var.cluster_state_bucket}"
+  team_name              = "${var.team_name}"
+  application            = "offender-management-allocation-manager"
+  is-production          = "${var.is-production}"
+  node_type              = "cache.m4.large"
+  environment-name       = "${var.environment-name}"
+  infrastructure-support = "${var.infrastructure-support}"
+  aws_region             = "eu-west-2"
```

Comment:
`aws_region` is deprecated. (replaced with the `providers` block)

Reply:
Done